turbot/resource_turbot_policy_setting.go

```diff
@@ -221,6 +245,14 @@ func resourceTurbotPolicySettingUpdate(d *schema.ResourceData, meta interface{})
 	input := mapFromResourceData(d, getPolicySettingUpdateProperties())
 	input["id"] = id
 
+	var err error
+	if value, ok := d.GetOk("template_input"); ok {
+		// NOTE: ParseYamlString expects a string
```

Comment:
// NOTE: ParseYamlString doesn't validate input as valid YAML format, on error it just returns the value